### DISCUSSION TITLE
Port Forward Proton VPN on Gluetun

### DISCUSSION THREAD

**Discussion by TENCHI125-PROTO** [qdm12/gluetun#2359] [Q&A] [answered]
Hi All,

I'm trying to follow Proton VPNs documentation for port forwarding, however I need to run natpmpc but it comes back as not found.

I setup Gluetun via CLI and OVPN

Does Gluetun need something extra adding to be able to run the command?

I am following this: https://protonvpn.com/support/port-forwarding-manual-setup/

Thanks,

**Comment by qdm12** [comment, marked as answer]
See https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/protonvpn.md#vpn-server-port-forwarding
